src/general/MainMenu.tscn

```diff
@@ -116,7 +116,7 @@ theme = ExtResource("1")
 script = ExtResource("6")
 ThriveLogoPath = NodePath("MenuContainers/LogoCenterContainer/Logo")
 MenuBackgrounds = [ExtResource("8"), ExtResource("7"), ExtResource("9")]
-Menu3DBackgroundScenes = ["res://assets/models/menu_backgrounds/MainMenuScene1.tscn", "res://assets/models/menu_backgrounds/MainMenuScene2.tscn", "res://assets/models/menu_backgrounds/MainMenuScene3.tscn"]
+Menu3DBackgroundScenes = ["res://assets/models/menu_backgrounds/MainMenuScene1.tscn"]
```

Comment:
This looks like an unintentional change...

Reply:
Yes, that was for testing.